packages/fluentui/docs-components/src/knobs/defaultComponents.tsx

```diff
@@ -35,11 +41,16 @@ const KnobBoolean: React.FunctionComponent<KnobComponentProps> = props => (
   />
 );
 
-const KnobNumber: React.FunctionComponent<KnobComponentProps> = props => (
+const KnobNumber: React.FunctionComponent<KnobNumberKnobComponentProps> = props => (
   <input
     onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
-      props.setValue(parseInt(e.target.value, 10));
+      let newValue = parseInt(e.target.value, 10);
+      newValue = newValue < props.min ? props.min : newValue > props.max ? props.max : newValue;
+      props.setValue(newValue);
```

Comment:
```suggestion
      props.setValue(parseInt(e.target.value, 10));
```

This looks redundant as `input` with `type=number` already handles this: https://www.w3schools.com/tags/tryit.asp?filename=tryhtml5_input_type_number

Reply:
It doesn't if the user actually type a bigger number than the max...